packages/jaeger-ui/tsconfig.lint.json

```diff
@@ -51,6 +51,7 @@
     "src/reducers/trace.js",
     "src/selectors/dependencies.js",
     "src/utils/configure-store.js",
+    "src/utils/parseQuery.ts",
```

Comment:
Why is this needed? Don't we pick up `*ts` files for linting automatically? If we don't it's a very sad state - linting should be on by default.

Reply:
Not actually. We aren't picking up *ts files automatically.

Without this rule:
![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/11b2f706-70ac-4bae-93ae-c7da4c14f51f)

Reply:
let's rename the file to tsx then, for now. I think we need to fix this separately.

Reply:
Done!

Reply:
Though, it can also be configured in tsconfig.json

```diff
{
    "extends": "../../tsconfig",
-   "include": ["src/**/*.tsx", "typings"]
+  "include": ["src/**/*.tsx", "src/**/*.ts", "typings"]
}